### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kyam-mayden**
Basic framework completed, additional page added for portfolio. Some content added, very little styling added at the moment.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+				<li><a href="#about">About</a></li>
+				<li><a href="portfolio.html" id="portFolio">Portfolio</a></li>
+				<li><a href="#foot">Contact</a></li>
+			</ul>
+			<div class="title">
+				<h1>Kyam Harris</h1>
+				<h4>Placeholder text, maybe skills and experience</h4>
+
+			</div>
+			<section class="social">
```
Comment (nsantiagoblair) [inline index.html]: Maybe make it another `ul` like your `.navlinks`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+				<a href="#" class="twitter"></a>
+				<a href="#" class="github"></a>
+				<a href="#" class="linkedIn"></a>
+			</section>
+			<div class="headImage">
```
Comment (nsantiagoblair) [inline index.html]: Maybe take out the text out the `<nav>` and put it in the `<header>`

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+.title h4{
+	color:#808080;
+}
+
+
+/*content*/
+
+main.container{
+	height:1300px;
```
Comment (nsantiagoblair) [inline styles.css]: can `height` be a `vh`?
also maybe try `min-height` attribute so you can set a height that can go bigger if content is too big for container.

File: index.html (hunk 1 above)
Comment (kyam-mayden) [reply index.html]: Completed

File: index.html (hunk 2 above)
Comment (kyam-mayden) [reply index.html]: A bit of a headache with styling, going to leave as it works atm

File: styles.css (hunk 3 above)
Comment (kyam-mayden) [reply styles.css]: changed to min-height

**Review by JamesCullen2 — APPROVED** [review]
Yes, all good !

**Pull request merged**